tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/SOAPWSBotTests.java

```diff
@@ -21,11 +21,11 @@
 @SuiteClasses({
 	SampleSoapServicesTest.class, // jaxws.spi.Provider not found
 	SOAPWSToolingIntegrationTest.class,
-	BottomUpWSTest.class,
-	TopDownWSTest.class,
-	WsClientTest.class,
-	WsTesterTest.class,
-	CxfWsClientTest.class
+	BottomUpWSTest.class, // PASS
+	TopDownWSTest.class, // PASS
+	WsClientTest.class, // PASS
+	WsTesterTest.class, // DH key exchange
+	CxfWsClientTest.class // exc. in UI thread
```

Comment:
@odockal I suppose you had these comment during debuging?

Reply:
That is right. though I would like to keep them for now.